src/validator/types/uri.rs

```diff
@@ -0,0 +1,30 @@
+#![allow(clippy::invalid_regex)]
```

Comment:
is this a bug in clippy ?

Reply:
Don't know (yet), have to investigate. Just pushed TODO there.